### PR TITLE
Home: Fixed responsiveness of image in statement section

### PR DESCRIPTION
**Pull request by Nikhil-Ladha**
Signed-off-by: Nikhil-Ladha <nikhilladha1999@gmail.com>

**Description**
Updated the designer image to remove the transparent background.
Fixed the responsive and added appropriate shadow at all the places where the image is used.
And, also aligned the "Proud Maintainer" header to the bottom left of the statement section. 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit da18eac385ec47f276eaad828e5ea520d928f7d3 at: https://6307b6e4ce26bd00673ade69--layer5.netlify.app

**Comment by leecalcote** [comment]
This is looking good. There is one small range between 1600px and 1612px that seems to be catching an overlap just yet - -

File: src/sections/Home/Proud-maintainers/proudMaintainers.style.js [hunk 1]
```diff
@@ -3,10 +3,16 @@ const ProjectItemWrapper = styled.section`
 	.left-col {
 		text-align: left;
 		align-self: flex-start;
-		margin-left: -5rem;
+		margin: -7rem auto auto 0;
```
Comment (Nikhil-Ladha) [inline src/sections/Home/Proud-maintainers/proudMaintainers.style.js:6]:
```suggestion
		margin: -5.5rem auto auto 0;
```

**Comment by Nikhil-Ladha** [comment]
> This is looking good. There is one small range between 1600px and 1612px that seems to be catching an overlap just yet -
> 
> Screen Shot 2022-08-25 at 12 58 40 PM

Resolved :)

**Comment by l5io** [comment]
🚀 Preview for commit 74025a0c10383d1fe07bfacb57570a720036a4f4 at: https://6307bd21ae02b905e260fa34--layer5.netlify.app

**Comment by leecalcote** [comment]
👏 thank you.

**Pull request merged**